release notes/v0.44.0.md

```diff
@@ -0,0 +1,221 @@
+
+### A new experimental module implementing the Web Crypto API specification [#2884](https://github.com/grafana/k6/pull/3007)
+
+This release includes a new experiment `webcrypto` module implementing (partially) the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) specification in k6.
+
+<details>
+<summary> Expand to see an example of the new functionality.</summary>
+
+An example shows encrypting and decrypting of a "Hello, World!" string using AES-CBC algorithm.
+
+```javascript
+import { crypto } from 'k6/experimental/webcrypto';
+
+export default async function () {
+  const key = await crypto.subtle.generateKey(
+    {
+      name: 'AES-CBC',
+      length: 256,
+    },
+    true,
+    ['encrypt', 'decrypt']
+  );
+
+  const encoded = stringToArrayBuffer('Hello, World!');
+  const iv = crypto.getRandomValues(new Uint8Array(16));
+
+  const ciphertext = await crypto.subtle.encrypt(
+    {
+      name: 'AES-CBC',
+      iv: iv,
+    },
+    key,
+    encoded
+  );
+
+  const plaintext = await crypto.subtle.decrypt(
+    {
+      name: 'AES-CBC',
+      iv: iv,
+    },
+    key,
+    ciphertext
+  );
+
+  console.log(
+    'deciphered text == original text: ',
+    arrayBufferToHex(plaintext) === arrayBufferToHex(encoded)
+  );
+}
+
+function arrayBufferToHex(buffer) {
+  return [...new Uint8Array(buffer)].map((x) => x.toString(16).padStart(2, '0')).join('');
+}
+
+function stringToArrayBuffer(str) {
+  var buf = new ArrayBuffer(str.length * 2); // 2 bytes for each char
+  var bufView = new Uint16Array(buf);
+  for (var i = 0, strLen = str.length; i < strLen; i++) {
+    bufView[i] = str.charCodeAt(i);
+  }
+  return buf;
+}
+```
+</details>
+
+The list of supported APIs and algorithms you could find in the project's [README](https://github.com/grafana/xk6-webcrypto/blob/v0.1.0/README.md#current-state)
+
+### Add sampling capabilities to the experimental tracing module [#2886](https://github.com/grafana/k6/pull/2886)
+
+This release adds sampling capabilities to the tracing module. You can now specify a sampling rate with `sampling` option when initializing a Client or in `tracing.InstrumentHTTP` function.
+
+### `browserContext.addCookies` [browser#760](https://github.com/grafana/xk6-browser/pull/760)
+
+Websites require cookies to help store session details which will be used by the backend to retrieve the state. For example, when a user logins into a website then the backend will store login session details in a cookie, which can be used in future actions performed on the website. For the given example, to speed up the test run, instead of having to first go through the login flow, it would be quicker to set a known valid cookie so that the user can easily test a different feature of the website. With this new feature cookies can be added to a `browserContext` and all new `page`s created from this `browserContext` will have the cookie assigned to them. Thanks @zucchinho for implementing the feature!
```

Comment:
I don't think we need to explain what cookies are, and how they're used :smile:

So I would shorten this to:
```suggestion
Cookies can now be added to a `BrowserContext` and all new `Page`s created from this context will have the cookie assigned to them. Thanks @zucchinho for implementing this feature!
```